ui/src/submissions/authors/schemas/author.js

```diff
@@ -28,6 +28,7 @@ const authorSchema = object().shape({
     .trim()
     .required()
     .label('Display Name'),
+  former_name: string().label('Alternate/former name'),
```

Comment:
`label` is used only in validation error messages, since `string()` without extra stuff like `required` won't cause any error ever, it's not necessary here.